src/Grpc.AspNetCore.HealthChecks/ServiceMappingCollection.cs

```diff
@@ -25,7 +25,7 @@ namespace Grpc.AspNetCore.HealthChecks;
 /// <summary>
 /// A collection of <see cref="ServiceMapping"/> used to map health results to gRPC health checks services.
 /// </summary>
-public sealed class ServiceMappingCollection : IEnumerable<ServiceMapping>
+public sealed class ServiceMappingCollection : ICollection<ServiceMapping>
```

Comment:
nit: would `IReadOnlyCollection<T>` be a better choice here? do you need mutators?